src/server/pegasus_server_impl.cpp

```diff
@@ -1375,6 +1414,7 @@ ::dsn::error_code pegasus_server_impl::start(int argc, char **argv)
         _meta_store->set_data_version(PEGASUS_DATA_VERSION_MAX);
         _meta_store->set_last_flushed_decree(0);
         _meta_store->set_last_manual_compact_finish_time(0);
+        _meta_store->set_usage_scenario(ROCKSDB_ENV_USAGE_SCENARIO_NORMAL);
```

Comment:
Defautly, the `get_envs` will show `null` result when using `shell`,  if here `set_usage_scenario(ROCKSDB_ENV_USAGE_SCENARIO_NORMAL)` at  first `start`, does that mean the `get_envs` will show `normal` after reopen when using `shell`?

Reply:
It depends on where we read data from when using pegasus_shell `get_app_envs`.
I think we will read data from meta server, so we would still get an empty string.